heartbeat/IPaddr2

```diff
@@ -701,6 +703,23 @@ log_arp_sender() {
     ocf_log $LOGLEVEL "$output"
 }
 
+# wrapper function to manage PID file to run arping in background
+run_with_pidfile() {
+    local cmdline
+    local pid
+    local rc
+
+    cmdline="$@"
+
+    $cmdline &
+    pid=$!
+    echo "$pid" > $SENDARPPIDFILE
+    wait $pid
```

Comment:
This is to create and remove the pid file of arping as similar manner  as the send_arp binary does.
Running it in the background once is to store the pid of arping via $! to the pid file (which could be used later to clean up if it's still running when arp_bg=true), and waiting it here is to remove the pid file as soon as arping exits at the next line.